### DISCUSSION TITLE
Feature request: multiple reporters

### DISCUSSION THREAD

**Discussion by OliverJAsh** [biomejs/biome#4761] [Ideas]
We're using the GitHub reporter (`--reporter=github`) which works great, however it would be useful if the lint errors were also reported in the terminal, like they are with the default summary reporter.

https://biomejs.dev/reference/reporters/

**Comment by weyert** [comment]
Yeah, or the ability to write the response to disk. E.g. I would like to run `github` reporter together with `junit` and have it save the `junit` output to disk at a location like `coverage/junit.xml`

**Comment by goleary** [comment]
I'd love to have this to get the standard output + the summary output without having to run biome multiple times.

Or give me a reporter that combines the standard and summary.

**Comment by llimllib** [comment]
I just hit this issue and had to insert a duplicate biome run to get output in the console, this would be great.

Another solution would be if the github reporter just also included the default reporting on stdout

**Comment by Netail** [comment]
Will check what I can do to get this feature in :)

**Comment by Netail** [comment]
A PR for multi reporters support has landed into the `next` branch, which will be the next minor release (version 2.4)

**Reply by llimllib** [reply]
just linking it: https://github.com/biomejs/biome/pull/8621
